File: CoreJava/student-app/src/com/techlabs/student/Student.java
package com.techlabs.student;

import java.io.*;
import java.text.*;
import java.util.Date;
import java.util.UUID;


@SuppressWarnings("serial")
public class Student implements Serializable {
	private long id;
	private String name;
	private int age;
	private String location;

	public Student() {
	}

	public Student(String name, int age, String location) {
		this.id = UUID.randomUUID().toString().replace("-", "");
		this.name = name;
		this.age = age;
		this.location = location;
	}

	public Student(long id, String name, int age, String location) {
		this.id = id;
		this.name = name;
		this.age = age;
		this.location = location;
	}

	public long getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public int getAge() {
		return age;
	}

	public String getLocation() {
		return location;
	}
}

File: DotNET/C#/StudentSerializeApp/StudentSerializeApp/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace StudentSerializeApp
{
    [Serializable]
    class Student
    {
        private long _id;
        private String _name;
        private int _age;
        private String _location;

        public Student(long id, String name, int age, String location)
        {
            _id = id;
            _name = name;
            _age = age;
            _location = location;
        }

        public long Id
        {
            get
            {
                return _id;
            }
        }

        public String Name
        {
            get
            {
                return _name;
            }
        }

        public int Age
        {
            get
            {
                return _age;
            }
        }

        public String Location
        {
            get
            {
                return _location;
            }
        }



    }
}
